src/test/java/org/opensearch/security/sanity/tests/SecurityRestTestCase.java

```diff
@@ -45,6 +45,8 @@ public class SecurityRestTestCase extends OpenSearchRestTestCase {
 
     private static final String CERT_FILE_DIRECTORY = "sanity-tests/";
 
+    static boolean useAdminAsPassword = false;
```

Comment:
yes, It is required by buildclient to establish what password to be used for building an admin user client. Since build client is invoked before test-setup, the only way to access it is via static variable (modified via @BeforeClass)